Detections/AzureDevOps/AzDOHistoricServiceConnectionAdds.yaml

```diff
@@ -0,0 +1,44 @@
+id: 5efb0cfd-063d-417a-803b-562eae5b0301
+name: Azure DevOps Service Conection Addition/Abuse - Historic Allowlist
+description: |
+  'This detection builds a allowlist of historic service connection use by Builds and Releases and compares to recent history, flagging growth of service connection use in non manually allowlisted, non historically allowlisted Build/Release runs.
+  This is to determine if someone is hijacking a build/release and adding many service connections in order to abuse or dump credentials from service connections.'
+severity: High
+queryFrequency: 2h
+queryPeriod: 14d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Execution
+  - Impact
+  - Escalation
```

Comment:
Please use the MITRE ATT&CK full names with no spaces. Here Escalation would be PrivilegeEscalation, there is no Impact so will need to remove or change to another and we also require Techniques in Detections - https://attack.mitre.org/techniques/enterprise/

Format as follows -

relevantTechniques:

T1078
T1110

Reply:
fixed in commit